deployments.go

```diff
@@ -77,6 +77,34 @@ func (s *deployerService) Deploy(ctx context.Context, db *gorm.DB, opts DeployOp
 	if err := json.NewEncoder(opts.Output).Encode(&msg); err != nil {
 		return r, err
 	}
-
+	go s.streamEvents(opts)
```

Comment:
I think the io.Writer will be closed after this method returns, so we'd probably want to block here until the events channel is closed.